### PR TITLE
Add proper emoji support

### PR DESCRIPTION
**Pull request by paked**
This PR adds emoji support to the Markdown renderer, using [gemoji](https://github.com/gmac/gemoji-parser#tokenizing).

I've added one test, but am happy to add more if wanted.

### CODE REVIEW

File: test/services/markdown_service_test.rb [hunk 1]
```diff
@@ -9,6 +9,14 @@ class MarkdownServiceTest < ActiveSupport::TestCase
     assert_equal(output, expected)
   end
 
+  test 'it parses emoji' do
```
Comment (paked) [inline test/services/markdown_service_test.rb:12]: Is this a false positive? Happy to fix, but the other tests in this file make the same 'mistake'

Comment (maxwofford) [inline test/services/markdown_service_test.rb:12]: I just set up houndci today so I have to send in a PR to make the rest of lecture-hall follow it's conventions. Could you make your PR follow it?

Comment (paked) [inline test/services/markdown_service_test.rb:12]: Sure!

**Comment by maxwofford** [comment]
I just want to take the time to say that you've made favorite test in all of lecture-hall @paked.

**Comment by zachlatta** [comment]
I added some changes on top of this and merged it over in #38. Changes are live now, thanks @paked :-).

**Pull request closed without merging**